index.js

```diff
@@ -17,7 +17,12 @@ express()
   .use(express.static(path.join(__dirname, 'public')))
   .set('views', path.join(__dirname, 'views'))
   .set('view engine', 'ejs')
-  .get('/', (req, res) => res.render('pages/index'))
+  .get('/', jsonParser, async (req, res) => {
+    try {
+      const client = await pool.connect();
+      const result = await client.query(`SELECT * FROM fridge_products`);
```

Comment:
Here's an updated query that you can use here instead of `SELECT * FROM fridge_products` that will let you also access the product name and type (since those only exist in the product table):
```
const result = await client.query(`SELECT prod_name, type, exp_dt, qty FROM fridge_products NATURAL JOIN products`);
```

Reply:
Thank you!